.github/workflows/ci.yaml

```diff
@@ -0,0 +1,68 @@
+name: CI
+on: [push]
```

Comment:
I think this might be better as:

```yml
on:
  pull_request:
  push:
    branches: [dev, staging, master]
```

The above means it will run on all PRs opened and pushes to the dev, staging and master

Reply:
I think you could also delete the circleci folder too so it stop running as we will only use the github ci once done

Reply:
I was planning to delete the circleci folder once I'd finished referring to it - it's quite a useful reference for setting up the github actions for now.

Reply:
Should we also delete .gitlab-ci related files?